Dockerfile.builder

```
@@ -59,3 +57,13 @@ RUN wget --quiet -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | ap
     apt-get update && \
     apt-get install --no-install-recommends -y -q \
     google-chrome-stable ca-certificates
+
+LABEL \
+        io.k8s.description="This is a component of OpenShift Container Platform and provides a web console." \
```

Copy/paste of what we have in our main Dockerfile.  I don't know that we need all of this, for the builder image, maybe just the maintainer and license are enough.

---

I realize this was a copy/paste, but maybe...

```suggestion
        io.k8s.description="The web console for the OpenShift Container Platform." \
```

---

Yeah, wasn't sure since this is the builder image that doesn't actually go into the product, do we need this at all?